src/CodeGen_Hexagon.cpp

```diff
@@ -1286,6 +1286,7 @@ string CodeGen_Hexagon::mattrs() const {
     if (target.has_feature(Halide::Target::HVX_128)) {
         attrs << ",+hvx-double";
     }
+    attrs.seekp(0, std::ios_base::end);
```

Comment:
@abadams @dsharletg : I agree. I am sorry I posted this patch a little too soon. Just as soon as I posted the PR,  I realized I could have done it better.

@dsharletg: In fact, we werent always using +hvx because we used that in the constructor and then promptly overwrote it with ", +long-calls" for hvx_64. Only for HVX_128, we used to have "+hvx,+hvx-double".